CONTRIBUTING.md

```diff
@@ -73,6 +73,27 @@ The subcategory object should follow this pattern
 > - Your entries should follow the above structure.
 
 --- 
+## Issues
+* Select an issue template from [issues](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.
```

Comment:
```suggestion
* Select an issue template from [issues](https://github.com/rupali-codes/LinksHub/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.
```